MC/bin/o2dpg_sim_workflow.py

```diff
@@ -519,7 +519,7 @@ def getDPL_global_options(bigshm=False, ccdbbackend=True):
 
 # preproduce the collision context / timeframe structure for all timeframes at once
 precollneeds=[GRP_TASK['name']]
-NEventsQED=10000  # max number of QED events to simulate per timeframe
+NEventsQED=int(INTRATE*0.3)  # max number of QED events to simulate per timeframe
```

Comment:
So at 50kHz, this would be 15000 which seems manageable. 
Could we have an explanation added as comment where this comes from?

Reply:
Is the comment enough? Otherwise I can go more in detail